InfyOmLabs/laravel-generator
# Publish User Crud with default fields

### What Implemented? 

- User CRUD publish with default field which are provided by Laravel framework.
-  Publish User CRUD via using `php artisan infyom.publish:user` command

**following files generated by this command**

1. Controller
2. Repository
3. Views
4. Route
5. Menu
6. Requests

before fire this command need to complete installation steps and publish layouts

after publish user CRUD. 

for more detail about issue.
How to integrate Users with scaffold? [#666](https://github.com/InfyOmLabs/laravel-generator/issues/666)

#### How to Test?
Add this package in. composer and update it. do all step of infyom pkg installations. 
```
 "infyomlabs/laravel-generator": "dev-issue-666-user-publish",
  "infyomlabs/adminlte-templates": "dev-issue-666-user-crud-template",
```
then after run command   `php artisan infyom.publish:user`
Check admin panel. you can see user menu and test it.
